recipes/libjxl/meta.yaml

```diff
@@ -0,0 +1,134 @@
+{% set version = "0.7.0" %}
+
+package:
+  name: libjxl-split
+  version: {{ version }}
+
+source:
+  - url: https://github.com/libjxl/libjxl/archive/v{{ version }}.tar.gz
+    sha256: 3114bba1fabb36f6f4adc2632717209aa6f84077bc4e93b420e0d63fa0455c5e
+    patches:
+      # Fix https://github.com/libjxl/libjxl/pull/582
+      - 0001-CHECK_EXCLUDE_LIBS.patch
+      # OpenEXR doesn't have pkg-config files in win64
```

Comment:
we could probably add one https://github.com/conda-forge/openexr-feedstock/issues/27

Reply:
I can try to push a fix

Reply:
Ok, it should be available in a few hours.